src/main/kotlin/uk/gov/justice/hmpps/offendersearch/config/HealthInfo.kt

```diff
@@ -10,15 +10,10 @@ import org.springframework.stereotype.Component
 import org.springframework.web.client.RestClientException
 
 @Component
-class HealthInfo : HealthIndicator {
+class HealthInfo(@param:Autowired(required = false) private val buildProperties: BuildProperties?) : HealthIndicator {
```

Comment:
can be required and not nullable since when running in intellij using gradle these days

Reply:
thanks